File: js_coding_q_and_a/07.js
// Find the factorial of a number.

const findFactorial = (n) => {
  var res = 1;
  while (n > 0) {
    res = res * n;
    n = n - 1;
  }
  return res;
};

// using recursion
const findFactorial1 = (n) => {
  if (n > 0) {
    return n * findFactorial1(n - 1);
  } else {
    return 1;
  }
};

console.log(findFactorial1(5));

File: py_coding_q_and_a/07.py
# // Find the factorial of a number.

def factorial(n):
    res=1
    while n>0:
        res=res* n
        n=n-1
    return res

def factorial1(n):
    if n>0 :
        return n * factorial1(n-1)
    else:
        return 1

print(factorial(5))
print(factorial1(5))